javalin/src/main/java/io/javalin/http/JavalinServletHandler.kt

```diff
@@ -64,6 +64,9 @@ class JavalinServletHandler(
     /** Indicates if exception occurred during execution of a tasks chain */
     private var errored = false
 
+    /** Indicates if following stages should be skipped, since request was redirected */
+    private var redirected = false
```

Comment:
Effectively, this is equal to the `termination` like I said on Discord and I think it would make more sense in a context of servlet handler to call it like that